### PR TITLE
make GIR report more robust

### PR DESCRIPTION
**Pull request by biyeun**
The usual find duplicate domain and delete method for fixing the broken GIR report seems to be failing. Instead I'm going to catch the `IntegrityError` in a `SoftAssert` when generating the report and allow the report generation to (hopefully) continue for the remaining domains.

addresses [HI-732](https://dimagi-dev.atlassian.net/browse/HI-732)

### CODE REVIEW

**Comment by esoergel** [comment]
Did you mean to close this?  Are you making a different PR?

**Comment by biyeun** [comment]
@esoergel re-opened

**Comment by biyeun** [comment]
I wasn't sure i was needing to do this...but I think this will fix the issue with the GIR running and failing hard

File: corehq/apps/data_analytics/gir_generator.py [hunk 1]
```diff
@@ -53,9 +60,21 @@ def fetch_latest_doc(dom1_id, dom2_id):
...
-        GIRRow.objects.bulk_create(
-            [GIRRow(**gir_row) for gir_row in rows]
-        )
+
+        final_rows = []
+        for row_data in rows:
+            try:
+                gir_row = GIRRow(**row_data)
+                final_rows.append(gir_row)
+            except IntegrityError:
```
Comment (esoergel) [inline corehq/apps/data_analytics/gir_generator.py]: Wouldn't this be raised only when it's attempted to be sent to the DB, ie in the `bulk_create` below?

**Pull request closed without merging**